### PR TITLE
fix orderBy arguments

### LINKED ISSUE #77: orderBy argument has no effect
The two `orderBy` tests in `e2e.spec.js` dont fail, if you switch `NAME_DESC` to `NAME_ASC` (and vice versa).

I noticed, that the actual `Collection.find` call in `query.js` gets called with

```
{
  sort: 'NAME_DESC'
}
```
instead of 
```
{
  sort: { name: -1 }
}
```

It seems like the order keys somehow don't get resolved into the corresponding mongoose option.
If you can give me a bump in the right direction, I'd be happy to fix this.

### PR DESCRIPTION
**Pull request by tothandras**
@jkempff, this should fix the orderBy arguments. Take a look at it! ;)
fixes #77

### CODE REVIEW

File: src/e2e.spec.js [hunk 1]
```diff
@@ -311,6 +311,24 @@ describe('e2e', () => {
           { node: { name: 'Foo' } },
           { node: { name: 'Mother' } }
         ]);
+
+        result = await graphql(schema, `{
```
Comment (jkempff) [inline src/e2e.spec.js]: I'd split this up in another test

Comment (tothandras) [inline src/e2e.spec.js]: oh, right :) will do

File: src/e2e.spec.js [hunk 2]
```diff
@@ -311,6 +311,24 @@ describe('e2e', () => {
...
+       }`);
+
+        expect(result.data.viewer.users.edges).to.be.eql([
+          { node: { name: 'Mother' } },
+          { node: { name: 'Foo' } },
+          { node: { name: 'Bar' } }
+        ]);
       });
 
       it('should return results in descending order', async function Test() {
```
Comment (jkempff) [inline src/e2e.spec.js]: I'd rename this one in something like "should be able to limit the ordered results"

**Comment by jkempff** [comment]
The code itself looks good, and it's working :)

**Comment by tothandras** [comment]
Great, thanks! :) I will fix the tests and merge it today.

**Comment by jkempff** [comment]
Very nice! And thank you for the fast responses!

**Comment by tothandras** [comment]
@jkempff released: `5.0.3` ;)

**Pull request merged**